server/main.c

```diff
@@ -348,6 +353,23 @@ static process_rec *init_process(int *argc, const char * const * *argv)
     process->argc = *argc;
     process->argv = *argv;
     process->short_name = apr_filepath_name_get((*argv)[0]);
+
+#if AP_HAS_THREAD_LOCAL
+    {
+        apr_status_t rv;
+        apr_thread_t *thd = NULL;
+        if ((rv = ap_thread_main_create(&thd, process->pool))) {
+            char ctimebuff[APR_CTIME_LEN];
+            apr_ctime(ctimebuff, apr_time_now());
+            fprintf(stderr, "[%s] [crit] (%d) %s: %s failed "
+                            "to initialize thread context (%i), exiting\n",
+                            ctimebuff, stat, (*argv)[0], failed, rv);
```

Comment:
```suggestion
            fprintf(stderr, "[%s] [crit] (%d) %s: Failed "
                            "to initialize thread context, exiting\n",
                            ctimebuff, rv, (*argv)[0]);
```
As otherwise we take over the stuff `apr_app_initialize`which seems wrong.

Reply:
Bad copypasta, will fix thanks.

Reply:
Fixed in r1897680 (a31b463532e83d557a4898cfb6411e144bc39266) and cheery picked here..